addon/webextension/manifest.json.template

```diff
@@ -30,8 +30,11 @@
       ]
     }
   ],
-  "icons": {
-    "32": "icons/icon-v2.svg"
```

Comment:
The [`icons` key](https://developer.mozilla.org/Add-ons/WebExtensions/manifest.json/icons) shouldn’t be removed, as it’s necessary for the icon to show up in `about:addons`.

Reply:
[The docs say this can be an object or string](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/page_action).  And changing it doesn't make a difference for `about:addons`.  @ExE-Boss Did you see a difference?

Reply:
That’s because this extension is currently a bootstrapped extension with an embedded WebExtension, so the icon of the bootstrapped extension is used as a fallback, but this should still be kept once this extension becomes a full WebExtension (see [bug 1422437](https://bugzil.la/1422437 "1422437 - Make Screenshots a regular WebExtension")).